### PR TITLE
POLAR-362 Update deprecated boolean of movable parameter to enum value

### PR DESCRIPTION
**Pull request by Softwarecules**
## Summary

Use of `movable` with `boolean` has been deprecated - The client configs and the examples have been updated to reflect this and now use an enum value as default.

## Instructions for local reproduction and review
Run the examples for `dish`, `afm` and `meldemichel` respectively and examine the pin-behaviour. It should work as intended and be dragable.

## Relevant tickets, issues, et cetera
#145 

Blocked by #147

### CODE REVIEW

File: packages/plugins/Pins/README.md [hunk 1]
```diff
@@ -17,7 +17,7 @@ The usage of `displayComponent` has no influence on the creation of Pins on the
...
 | coordinateSource | string? | The pins plugin may react to changes in other plugins. This specifies the path to such store positions. The position must, when subscribed to, return a GeoJSON feature. Please mind that, when referencing another plugin, that plugin must be in `addPlugins` before this one. |
 | initial          | initial? | Configuration options for setting an initial pin. |
-| movable          | boolean? \| 'drag' \| 'click' \| 'none' | Whether a user may drag and re-click the pin (`drag` or `true`), only re-click it (`click`) or may only be placed programmatically (`none` or `false`). Defaults to 'none'. **Using a boolean for this configuration has been deprecated and will be removed in the next major release.**           |
+| movable          | 'drag' \| 'click' \| 'none' | Whether a user may drag and re-click the pin (`drag` or `true`), only re-click it (`click`) or may only be placed programmatically (`none` or `false`). Defaults to 'none'. **Using a boolean for this configuration has been deprecated and will be removed in the next major release.**           |
```
Comment (dopenguin) [inline packages/plugins/Pins/README.md:20]:
```suggestion
| movable          | boolean \| 'drag' \| 'click' \| 'none'? | Whether a user may drag and re-click the pin (`drag` or `true`), only re-click it (`click`) or may only be placed programmatically (`none` or `false`). Defaults to 'none'. **Using a boolean for this configuration has been deprecated and will be removed in the next major release.**           |
```
The boolean should still be present, as it is still a valid configuration as of now.  
I also moved the `?` (indicator for optionality) to the end.

**Review by dopenguin — CHANGES_REQUESTED** [review]
Added a required change to the README.md and also suggested some slight adjustments for the changelog entries.

🏓 @Softwarecules

**Comment by Softwarecules** [comment]
> Added a required change to the README.md and also suggested some slight adjustments for the changelog entries.
> 
> 🏓 @Softwarecules

Applied the suggested changes :) 
🏓 @dopenguin

**Review by dopenguin — COMMENTED** [review]
Approve inbound as soon as #147 has been merged to main @Softwarecules

**Comment by Softwarecules** [comment]
> Approve inbound as soon as #147 has been merged to main @Softwarecules

@dopenguin #147 merged 🫡

**Review by dopenguin — APPROVED** [review]
🚀 @Softwarecules

**Pull request merged**